test/Ocelot.UnitTests/Responder/HttpContextResponderTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.AspNetCore.Http;
+using Moq;
+using Ocelot.Headers;
+using Ocelot.Responder;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using TestStack.BDDfy;
+using Xunit;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Ocelot.UnitTests.Responder
+{
+    public class HttpContextResponderTests
+    {
+        readonly HttpContextResponder _responder;
+
+        readonly Mock<IRemoveOutputHeaders> _removeOutputHeaders;
+
+        HttpContext _httpContext;
+
+        HttpResponseMessage _httpResponseMessage;
+
+        public HttpContextResponderTests()
+        {
+            _removeOutputHeaders = new Mock<IRemoveOutputHeaders>();
+            _httpContext = new DefaultHttpContext();
+            _httpResponseMessage = new HttpResponseMessage();
+            _httpResponseMessage.Content = new MyHttpContent();
+            _responder = new HttpContextResponder(_removeOutputHeaders.Object);
+        }
+
+        [Fact]
+        public void DoSomething()
```

Comment:
oh I should have deleted that file - this is what I started writing before I realised why you're using the test server. I'll remove it.